src/octoprint/timelapse.py

```diff
@@ -758,6 +762,7 @@ def _perform_capture(self, filename, onerror=None):
                 stream=True,
                 timeout=self._snapshot_timeout,
                 verify=self._snapshot_validate_ssl,
+                auth=HTTPDigestAuth(self._snapshot_auth_user,self._snapshot_auth_pass)
```

Comment:
I wonder if this probably shouldn't be added if there is no username/password setup, to avoid any potential issues with the majority of people that don't have authentication. If there is nothing extra sent when username/password are not enabled then it *might* be OK.

Reply:
If the URL does not request a username or password passing it doesn't do anything